ui/lib/core/addon/templates/components/ttl-picker2.hbs

```diff
@@ -1,30 +1,37 @@
-<Toggle
-  @name={{this.label}}
-  @status="success"
-  @size="small"
-  @onChange={{action "toggleEnabled"}}
-  @checked={{this.enableTTL}}
-  data-test-ttl-toggle
->
-  <span class="ttl-picker-label is-large">{{this.label}}</span><br />
-  <div class="description has-text-grey">
+{{#if this.hideToggle}}
+  <span class="ttl-picker-label">{{this.label}}</span><br />
+  <div class="sub-text">
     <span>{{this.helperText}}</span>
-    {{#if this.description}}
```

Comment:
Would it be worth keeping the tooltip option here if `hideToggle` is true for when there is a description? Might be useful down the road if this part of the component is used outside of MFA

Reply:
Good idea. 👍